src/pages/index.js

```diff
@@ -119,44 +125,78 @@ function handleOverlayClick(evt) {
   }
 }
 
-function openModal(modal) {
-  modal.classList.add("modal_opened");
-  document.addEventListener('keydown', handleEscKey);
-  modal.addEventListener('mousedown', handleOverlayClick);
-};
-
-function closeModal(modal) {
-  modal.classList.remove("modal_opened");
-  document.removeEventListener('keydown', handleEscKey);
-  modal.removeEventListener('mousedown', handleOverlayClick);
-};
-
-function resetCardForm() {
-  cardFormElement.reset();
-  cardModalSubmitBtn.disabled = true;
-  cardModalSubmitBtn.classList.add("modal__submit-btn_inactive");
+function makeEditRequest() {
+  return api.editUserInfo({
+    name: editModalNameInput.value,
+    about: editModalDescriptionInput.value
+  }).then((userInfo) => {
+    profileName.textContent = userInfo.name;
+    profileDescription.textContent = userInfo.about;
+  })
+  .then(() => closeModal(editModal))
+  .catch(console.error);
```

Comment:
You shouldn't add `catch` if you use `handleSubmit` becuase it contains `catch` inside it. This is the main purpose of `handleSubmit` not to duplicate the code in each submit handler

Reply:
Applied prior feedback from before implementation of the handleSubmit function.